tests/src/test/scala/cats/tests/AlternativeSuite.scala

```diff
@@ -2,6 +2,14 @@ package cats
 package tests
 
 class MonadCombineSuite extends CatsSuite {
```

Comment:
How about renaming the class as well? :)

Reply:
Oops, nice catch!